src/UI/Components/Diagram/Mocks/services.ts

```diff
@@ -825,6 +825,7 @@ export const testService: ServiceModel = {
   embedded_entities: [
     {
       name: "peerings",
+      type: "peerings",
```

Comment:
isn't the type supposed to be singular instead of plural? I know this is mock data, but wouldn't hurt to keep it right.

Reply:
My bad, changed